source/location_access/locacc_death_mountain.cpp

```diff
@@ -32,6 +30,12 @@ void AreaTable_Init_DeathMountain() {
                                       (IsAdult && CanPlantBean(DEATH_MOUNTAIN_TRAIL) &&
                                        (HasExplosives || GoronBracelet));
                            } }),
+            LocationAccess(DMT_ROCK_BLUE_RUPEE,
+                           { [] { return IsChild && Here(DEATH_MOUNTAIN_TRAIL, [] { return CanBlastOrSmash; }); } }),
```

Comment:
This boulder is not present in the adult scene so shouldn't use the "Here()" method
```suggestion
                           { [] { return IsChild && CanBlastOrSmash; } }),
```